libs/designsystem/src/lib/components/list/list.component.ts

```diff
@@ -234,7 +238,7 @@ export class ListComponent implements OnInit, AfterViewInit, OnChanges {
     args.event.stopPropagation();
   }
 
-  getItemEndClass(index: number, section?: any[]): EndClass {
+  getFirstOrLastClass(index: number, section?: any[]): EndClass {
```

Comment:
Consider popping on a `private`, `public` or prefix with `_`

Reply:
Also renamed to `_getBoundaryClass`